ops/terraform/implementation/backend.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>3.66.0"
+    }
+
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "= 2.41.0"
+    }
+    azapi = {
+      source  = "azure/azapi"
+      version = "= 1.8.0"
+    }
+
+    random = {
+      source  = "hashicorp/random"
+      version = "~>3.5.1"
+    }
+  }
+
+  backend "azurerm" {
+    storage_account_name = "tfstaterv2024"
```

Comment:
By doing this we will no longer be able to trigger our different environments via the individual config files. Are you thinking of ways to keep our infrastructure flexible at runtime? Unfortunately Terraform doesn't allow variables within `backend`.